app/models/ownership.rb

```diff
@@ -0,0 +1,22 @@
+class Ownership
+  include OwnershipClient
+  include ActiveModel::Model
+
+  # define_model_callbacks :create
+  # before_create :create_remote_ownership
+
+  validates :model_id,   presence: true, length: { maximum: 36 }
+  validates :model_type, presence: true
+  validates :owner_id,   presence: true
+
+  # obj = {:model_id => SecureRandom.uuid, :model_type => 'bio', :owner_idd => 'hc6@sanger.ac.uk'}
+  def create_remote_ownership(obj)
+  	post(obj)
+  end
+
+  # obj = 'b7a32344-cd0b-4b46-b986-48c1888c99a1'
+  def get_remote_ownership(obj)
+  	get(obj)
+  end
```

Comment:
get_remote_ownership has to be a class method, instead of an instance method, and needs to return an instance of Onwership class. This is because, when you look for an ownership by uuid you don't have an instance yet, you only have the name of the class. Same thing while creating a new ownership (create_remote_ownership).
To do it you can change it to something like:

def self.get_remote_ownerhip(uuid)
  new(some_filter(get(uuid)))
end